zio-http/src/main/scala/zhttp/experiment/HttpEndpoint.scala

```diff
@@ -26,8 +26,8 @@ case class HttpEndpoint[-R, +E](http: Http[R, E, Request, Response[R, E]]) { sel
     evE: E <:< Throwable,
   ): ChannelHandler =
     new ChannelInboundHandlerAdapter { ad =>
```

Comment:
Can we merge `HttpEndpoint` with `HttpApp`

Reply:
Done